broadcast/broadcast-client-mock/mock-consts/mock_consts.go

```diff
@@ -0,0 +1,23 @@
+package mock_consts
+
+const (
+	MockedProviderMain              = "https://mocked_arc_api_url.com/arc"
+	MockedProviderSecondary         = "https://secondary_mocked_arc_api_url.com/arc"
+	MockedMaxScriptSizePolicy       = 100000000
+	MockedMaxTxSigOpsCountPolicy    = 4294967295
+	MockedMaxTxSizePolicy           = 100000000
+	MockedMaxTxSizePolicySecondary  = 220000000
+	MockedMiningFeeBytes            = 1000
+	MockedSatoshisPerBytes          = 1
+	MockedSatoshisPerBytesSecondary = 2
+	MockedTimestamp                 = "2023-09-05T17:03:49.537230128Z"
+	MockedTimestampSecondary        = "2023-09-05T17:05:29.736256927Z"
+	MockedTxResponseStatus          = 200
+	MockedTxResponseTitle           = "OK"
+	MockedTxStatus                  = "SEEN_ON_NETWORK"
+	MockedTxBlockHash               = "123123"
+	MockedTxBlockHashSecondary      = "123123"
+	MockedTxBlockHeight             = 123456
+	MockedTxBlockHeightSecondary    = 123456
+	MockedTxExtraInfo               = ""
```

Comment:
Do we need this `Mocked` prefix in each and every const name?

Reply:
Done